main.go

```diff
@@ -491,6 +491,30 @@ func main() {
 			return err
 		}
 
+		//
+		// nomad-server
+		//
+		nomadServerDescription := "A Pulumi definition for deploying a cluster of Nomad servers as DigitalOcean droplets"
+		nomadServerRepositoryArgs := StandardRepositoryArgs("nomad-server", &nomadServerDescription)
+		nomadServer, err := github.NewRepository(ctx, "nomad-server", &nomadServerRepositoryArgs)
+		if err != nil {
+			return err
+		}
+		if err = MigrateDefaultBranchToMain(ctx, "nomad-server", nomadServer); err != nil {
```

Comment:
Use `RequireMainAsDefaultBranch` unless the repository already has a different default branch that needs renaming